### ISSUE TITLE
Expected a Content Page, but it's absent

### ISSUE THREAD

**Issue by phtan** [willnorris/roads-and-bridges#8]
Hey Will,

I appreciate your EPUB adaptation.

I am trying to quickly scan through the report so that I can give a quick digest of sorts, at a social event related to FOSS.

This means the ability to jump between chapters would be helpful to me. As it is, in my reader-app for the mobile OS Android, in the place where I usually see a list of hyperlinks of chapter titles, I see a blank space. My reader is called Overdrive.

Do you think my reader is mal-functioning, or is this list of hyperlinks missing from your EPUB?

I plan to provide the afore-mentioned quick digest, on 25th July this year. (This coming Thursday).

**Comment by willnorris** [comment]
I thought Overdrive was just for reading books borrowed from a library?  How do you even load arbitrary epubs into the app?  I was trying to reproduce you're issue, but am not even sure how to open the epub in the app.

Have you tried using a different app?

**Comment by phtan** [comment]
About how I open EPUB on my mobile phone - it isn''t always how I have done it, but I navigate to the directory with the EPUB, in a File Explorer-like app called Total Commander; then I click on it; then I have a choice of apps to open the EPUB with; I choose between Overdrive or other apps. 

Just now, I chose an app called Adobe Digital Editions instead of Overdrive.

I can now jump from one chapter to another, in the app Adobe Digital Editions. That is to say, I can see a 'globally' available Table of Contents, no matter which part of the e-book I happen to be in. (Just had to switch from Overdrive).

Closing this issue. Thanks for getting back to me so quickly.
